MonitorizareVot/Onboarding/Controllers/AddNotesViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AddNotesViewController.swift
+//  MonitorizareVot
+//
+//  Created by Geart Otten on 13/10/2019.
+//  Copyright © 2019 Code4Ro. All rights reserved.
+//
+
+import UIKit
+
+class AddNotesViewController: UIViewController {
+
+    // MARK: Variables
+    
+    private var onboardingView: OnboardingView!
+    
+    private var onboardingViewModel = OnboardingViewModel(title: "Add notes",
```

Comment:
I suggest not creating multiple view controllers if the only thing that changes is the model. Instead, create a single `OnboardingChildViewController` that will take a model as a parameter, then construct the list with different models, but same controller class.